src/index.ts

```diff
@@ -326,6 +332,28 @@ export async function install(
     };
     const packageBundle = await rollup.rollup(inputOptions);
     await packageBundle.write(outputOptions);
+
+    if (include && nomodule) {
+      // Resolve `/web_modules/` relative to package.json rather than system root
+      function renameImportPlugin() {
+        return {
+          name: 'rename-import-plugin',
+          resolveId(source) {
+            if (source.startsWith(path.sep)) {
+              return {id: source.replace(path.sep, path.join(cwd, path.sep))};
+            }
+            return null; // null means don’t rename
+          },
+        };
+      }
+      const noModuleBundle = await rollup.rollup({
+        input: glob.sync(include, {ignore: exclude})[0], // Note: this isn’t a good substitute for an entry file
```

Comment:
See other convo. Here, if we omit the `[0]`, it won’t work because it’ll try and export multiple bundles. We need to ask the user what their entry file is.